BrandingPolicyCheck.py

```diff
@@ -35,13 +35,6 @@ def check(self, pkg):
                 if (r[1] != rpm.RPMSENSE_EQUAL or not r[2][1].startswith('1')):
                     printError(pkg, 'suse-branding-unversioned-requires', r[0])
 
-        # verify that it uses branding conflicts
-        for r in pkg_conflicts:
-            if r.startswith("otherproviders("):
```

Comment:
Why not inverting this test an printing an info or warning that the old style 'otherproviders()' is being used

Reply:
Not everyone users spec cleaner everywhere, so the inverse warning is still useful

Reply:
Then it should be completely new check as it does not apply only for branding don't you think?

Reply:
Yep otherproviders is suseism to avoid rpm breaking on provides conflicting on the same namespace. It was fixed with rpm that we have in sle11 so it is no longer really needed.